apps/frontend/src/libs/types/components/link-type.type.ts

```diff
@@ -0,0 +1,3 @@
+type LinkType = "button" | "navLink";
```

Comment:
Looks like a type for link component only. Move to ~/libs/components/link/libs/types.ts